easybuild/easyconfigs/s/snakemake/snakemake-7.18.2-foss-2021b.eb

```diff
@@ -0,0 +1,96 @@
+easyblock = 'PythonBundle'
+
+name = 'snakemake'
+version = '7.18.2'
+
+homepage = 'https://snakemake.readthedocs.io'
+description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('GitPython', '3.1.24'),
+    ('IPython', '7.26.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('wrapt', '1.14.1', {
+        'checksums': ['380a85cf89e0e69b7cfbe2ea9f765f004ff419f34194018a6827ac0e3edfed4d'],
+    }),
+    ('ratelimiter', '1.2.0.post0', {
```

Comment:
ratelimiter has been replaced with throttler, so can be removed?

Reply:
@smoors fixed in 164eb8f226